guides/theming.md

```diff
@@ -267,6 +267,10 @@ styles than necessary.
 To control which theme applies when, `@include` the mixins only within a context specified via
 CSS rule declaration. See the [documentation for Sass mixins][sass-mixins] for further background.
 
+**Note:**, if your app's content is not placed inside a `mat-sidenav-container` element, you need
+to add the `mat-app-background` CSS class to your wrapper element (for example the `body`).
+This ensures that the proper theme background is applied to the page.
```

Comment:
I think this probably belongs in its own section rather than in the "Defining multiple themes" section.

I might write this like
```markdown
### Application background color

By default, Angular Material does not apply any styles to your DOM
outside of its own components. If you want to set your application's
background color to match the components' theme, you can either:
1. Put your application's main content inside `mat-sidenav-container`, assuming you're using `MatSidenav`, or
2. Apply the `.mat-app-background` CSS class to your main content root element (typically `body`).
```